lib/ruby/truffle/truffle/bigdecimal.rb

```diff
@@ -1,5 +1,15 @@
 # TODO require bigdecimal/*
 
+class Truffle::BigDecimal < Numeric
+  include Comparable
+
+  def ==(other)
+    (self <=> other) == 0
+  end
```

Comment:
`include`-ing Comparable should already do this, but it might have some differences regarding exceptions rescued.